content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -25,7 +25,21 @@ it succeeds, making sure those probes don't interfere with the application start
 This can be used to adopt liveness checks on slow starting containers, avoiding them
 getting killed by the kubelet before they are up and running.
 
+{{< caution >}}
+Liveness probes can be a powerful way to recover from application failures, but
+they should be used with caution. Liveness probes must be configured carefully
+to ensure that they truly indicate unrecoverable application failure, for example a deadlock.
+A common pattern for liveness probes is to use the same low-cost HTTP endpoint
+as for readiness probes, but with a higher failureThreshold. This ensures that the pod
+is observed as not-ready for some period of time before it is hard killed.
+{{< /caution >}}
 
+{{< note >}}
+Incorrect implementation of liveness probes can lead to cascading failures. This results in
+restarting of container under high load; failed client requests as your application became less
+scalable; and increased workload on remaining pods due to some failed pods.
+Understand the difference between readiness and liveness probes and when to apply them for your app.
+{{< /note >}}
```

Comment:
I would move this outside of a note callout. The caution callout is enough IMO.

Reply:
Hello @sftim , thanks for the review.

I put this statement into the note section because it describe about about possible pod and container failures.

Reply:
My take of @sftim's suggestion is that we already have a caution callout on line 32. This note callout doesn't add much value. We can put it as plain text. WDYT?

Reply:
@KanishkaVerma054 Do you think we can/should remove line 38 and 43 to avoid bombing users with many alerts, notes within a single page? The warning from line 32-36 is already a strong message.